osf/models/node.py

```diff
@@ -438,6 +438,11 @@ def is_preprint_orphan(self):
         return False
 
     @property
+    def has_published_preprint(self):
+        PreprintService = apps.get_model('osf.PreprintService')
+        return PreprintService.objects.filter(node_id=self.id, is_published=True).exists()
```

Comment:
@sloria Can you say more or point to an example, I'm not sure I understand the change.